src/OrchardCore.Modules/OrchardCore.OpenId/Drivers/OpenIdServerSettingsDisplayDriver.cs

```diff
@@ -120,6 +122,8 @@ public override async Task<IDisplayResult> UpdateAsync(OpenIdServerSettings sett
             new PathString("/connect/userinfo") : PathString.Empty;
         settings.IntrospectionEndpointPath = model.EnableIntrospectionEndpoint ?
             new PathString("/connect/introspect") : PathString.Empty;
+        settings.PushedAuthorizationEndpointPath = model.EnablePushedAuthorizationEndpoint ?
+            new PathString("/connect/par") : PathString.Empty;
```

Comment:
Note: I opted for `/connect/par` as `/par` is a quite common value, but I'm happy to change it if you have better ideas.